src/olympia/promoted/models.py

```diff
@@ -78,6 +78,20 @@ def approve_for_version(self, version):
                 version=version,
                 group_id=self.group_id,
                 application_id=app.id)
+        try:
+            del version.approved_for_groups
+        except AttributeError:
+            pass
+
+    def check_subscription_for_approval(self):
```

Comment:
Nit: This returns a boolean, but from the name it isn't clear (to me anyway), what True vs. False actually means.

Reply:
I tried to explain in the comment (just below).  Any suggestions for a better name?  (This is after a few iterations!)

Reply:
I think the big disconnect for me is that this is a boolean, so I think it would be clearer if that name was of the format `isXx` or `hasXx`. So maybe something like `hasPendingSubscription`?

Reply:
https://github.com/mozilla/addons-server/pull/15915/commits/a85ee788b2834b3504ba3c1f9b84679a014db9e2 - changing the logic around helped, 

> thanks!